image-definitions/kubeflow/components/example-notebook-servers/jupyter-pytorch/cuda-requirements.txt

```diff
@@ -0,0 +1,4 @@
+--find-links https://download.pytorch.org/whl/torch_stable.html
+torch
+torchvision
```

Comment:
`tourch` and `tourchvision` are set to point to latest to solve Critical CVE.
Could not do pinning to specific versions. `pip` failed to resolve dependency.